app/Platform/Models/Achievement.php

```diff
@@ -123,6 +123,13 @@ public function getCanonicalUrlAttribute(): string
         return route('achievement.show', [$this, $this->getSlugAttribute()]);
     }
 
+    public function getCanSendConnectUpdatesForUsers(User|string $user): bool
```

Comment:
This method name doesn't feel right - especially on the achievement. Maybe `getCanUnlockBeDelegatedByUser`?

Reply:
I definitely get what you're feeling here. I've renamed them in both `Game` and `Achievement`... maybe it's a little better now.

`Game::getCanDelegateActivity()`
`Achievement::getCanDelegateUnlocks()`